test/controllers/games_controller_test.rb

```diff
@@ -0,0 +1,53 @@
+require "test_helper"
+
+class GamesControllerTest < ActionDispatch::IntegrationTest
+  include ParsedResponse
+  include GameSupport
+  include GameAsserts
+  
+  test 'get index: successful' do
+    get_games_index
+
+    assert_response :success
+    assert_equal Game.all.size, response_data.size, 'Game - Index'
```

Comment:
Usa variables muy semánticas porque esta es la documentación de tu código y se un poco más descriptivo en la descripción, así como en el título, es decir, en vez de `'get index: successful'` un `'Valid Index Games'`

```suggestion
    get_games_index
    assert_response :success
    
    all_games_size = Game.all.size
    response_games_size = response_data.size
    assert_equal all_games_size, response_games_size, 'All records count in Index'
```